src/EspMQTTClient.h

```diff
@@ -165,7 +165,7 @@ class EspMQTTClient
   inline bool isConnected() const { return isWifiConnected() && isMqttConnected(); }; // Return true if everything is connected
   inline bool isWifiConnected() const { return _wifiConnected; }; // Return true if wifi is connected
   inline bool isMqttConnected() const { return _mqttConnected; }; // Return true if mqtt is connected
-  inline bool getConnectionEstablishedCount() const { return _connectionEstablishedCount; }; // Return the number of time onConnectionEstablished has been called since the beginning.
+  inline short getConnectionEstablishedCount() const { return _connectionEstablishedCount; }; // Return the number of time onConnectionEstablished has been called since the beginning.
```

Comment:
Interesting catch! It seems like the last change to that line was 2 years ago so it does not seem to be used that often. I wonder where its useful?

When looking at the variable itself it is an `unsigned int`. `bool` seems definitely wrong but I think it should be the same in all places.
https://github.com/plapointe6/EspMQTTClient/blob/d54a898b9a14d0c6e4f85db112029b198976aa23/src/EspMQTTClient.h#L92
```suggestion
  inline unsigned int getConnectionEstablishedCount() const { return _connectionEstablishedCount; }; // Return the number of time onConnectionEstablished has been called since the beginning.
```